ansible_collections/arista/avd/roles/eos_designs/python_modules/mlag/avdstructuredconfig.py

```diff
@@ -170,6 +170,9 @@ def port_channel_interfaces(self):
         if (self.shared_utils.fabric_sflow_mlag_interfaces) is not None:
             port_channel_interface["sflow"] = {"enable": self.shared_utils.fabric_sflow_mlag_interfaces}
 
+        if (flow_tracker := self.shared_utils.get_flow_tracker(None, "mlag_interfaces", None)) is not None:
+            port_channel_interface["flow_tracker"] = flow_tracker
+
```

Comment:
Since we use `strip_empties_from_dict` at the bottom, you can just add `"flow_tracker": self.shared_utils.get_flow_tracker(...)` above in the `port_channel_interface` dict initialization.